plugins/acss-kit/docs/recipes.md

```
@@ -194,40 +194,47 @@ An opt-in, static-HTML smoke check that exercises the *compiled CSS and `data-*`
    ```sh
-   python3 -m http.server 7743 &
+   tests/serve.sh
    ```
 
-   Background it so the shell stays usable. Stop it later with `kill %1` or by capturing the PID (`python3 -m http.server 7743 & echo $!`).
+   This watches SCSS and HTML files, recompiles CSS on save, and auto-reloads the browser. Prints the serving URL (`http://localhost:7743/` or next available port if 7743 is busy).
 
-4. Open the preview — manually in any browser, or in a Claude Code session via the `claude-in-chrome` MCP (`navigate` then `computer screenshot`):
+3. Open the preview — manually in any browser, or in a Claude Code session via the `claude-in-chrome` MCP (`navigate` then `computer screenshot`):
 
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)